.github/actions/code_build/action.yaml

```diff
@@ -0,0 +1,128 @@
+name: Code Build 
```

Comment:
I'm not sure this should be related to this PR; I'd leave this PR focues on the CDN deploy